source/reference/write-concern.txt

```diff
@@ -158,6 +158,12 @@ the write operation has been written to the :doc:`journal
      configuration setting determines the behavior. See
      :ref:`wc-ack-behavior` for details.
 
+   - The :doc:`WiredTiger </core/wiredtiger>` and :doc:`MMAPv1 </core/mmapv1>`
+     storage engines handle setting a write concern that includes ``j: true``
```

Comment:
This sentence makes it sound like storage engines set a write concern. 

I think we can just simplify and get to the meat:

- For a write concern that includes ``j: true``,

  - :doc:`WiredTiger </core/wiredtiger>` forces an immediate, synchronous journal flush.
  - :doc:`MMAPv1 </core/mmapv1>`  waits for the next scheduled journal flush.